pandas/core/indexes/datetimes.py

```diff
@@ -1937,15 +1937,18 @@ def tz_convert(self, tz):
                      mapping={True: 'infer', False: 'raise'})
     def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         """
-        Localize tz-naive DatetimeIndex to given time zone (using
-        pytz/dateutil), or remove timezone from tz-aware DatetimeIndex
+        DatetimeIndex.tz_localize used for Localize tz-naive DatetimeIndex.
+
+	tz_localize method is use to convert naive DatetimeIndex into any other localize
+	DatetimeIndex 
+
 
         Parameters
         ----------
         tz : string, pytz.timezone, dateutil.tz.tzfile or None
             Time zone for time. Corresponding timestamps would be converted to
             time zone of the TimeSeries.
-            None will remove timezone holding local time.
+            None will remove timezone holding local time
```

Comment:
Please leave this final '.' (the updated validation script will also give a message about this)